### PR TITLE
Extract class refactoring: Extracting email generator to a different …

### PR DESCRIPTION
**Pull request by almasfiza**
…class from internet

### CODE REVIEW

File: src/main/java/net/datafaker/providers/base/BaseFaker.java [hunk 1]
```diff
@@ -423,4 +423,6 @@ public static Method getMethod(AbstractProvider<?> ap, String methodName) {
         if (ap == null || (map = METHODS.get(ap.getClass())) == null) return null;
         return map.get(methodName);
     }
+
+
```
Comment (kingthorin) [inline src/main/java/net/datafaker/providers/base/BaseFaker.java:427]: This seems unnecessary.

**Review by kingthorin — COMMENTED** [review]
This seems to be a breaking change. Since the methods are no longer available through Internet.

**Comment by bodiam** [comment]
This whole PR seems unnecessary. Sorry, closing.

**Comment by kingthorin** [comment]
I shouldn’t have said “breaking” I should have said “binary incompatible” or “changes the public API” 🤷‍♂️ 

But yeah, anyway I agree I’m not sure this added/helped. That’s not meant to discourage you, feel free to ask questions in the issue you opened if you’re not sure about any of the changes you think of.

**Comment by almasfiza** [comment]
> I shouldn’t have said “breaking” I should have said “binary incompatible” or “changes the public API” 🤷‍♂️
> 
> But yeah, anyway I agree I’m not sure this added/helped. That’s not meant to discourage you, feel free to ask questions in the issue you opened if you’re not sure about any of the changes you think of.

Yea! I understand. Thank you :)

**Pull request closed without merging**